core/src/test/java/GameObjects/UrlBuilderImpTest.java

```diff
@@ -0,0 +1,95 @@
+package GameObjects;
+
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.runners.MockitoJUnitRunner;
+
+import static org.junit.Assert.assertEquals;
+import static org.mockito.Mockito.when;
+
+/**
+ * Tests for the UrlBuilder. The URLs provided were taken directly from the database using their
+ * API.
+ */
+@RunWith(MockitoJUnitRunner.class)
+public class UrlBuilderImpTest {
```

Comment:
What do we want the urlBuilder to throw in the case that it receives these unexpected values?